### PR TITLE
[DBTV] Add support for articles (Closes #8653)

### LINKED ISSUE #8653: Support Dagbladet (Norwegian newspaper)
Request support for downloading videos from Dagbladet (Norwegian newspaper):

```
~/$ youtube-dl -v http://www.dagbladet.no/2016/02/23/nyheter/nordlys/ski/troms/ver/43254897/
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v', u'http://www.dagbladet.no/2016/02/23/nyheter/nordlys/ski/troms/ver/43254897/']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.02.10
[debug] Python version 2.7.11 - Darwin-15.3.0-x86_64-i386-64bit
[debug] exe versions: ffmpeg 2.8.3, ffprobe 2.8.3, rtmpdump 2.4
[debug] Proxy map: {}
[generic] 43254897: Requesting header
WARNING: Falling back on generic information extractor.
[generic] 43254897: Downloading webpage
[generic] 43254897: Extracting information
ERROR: Unsupported URL: http://www.dagbladet.no/2016/02/23/nyheter/nordlys/ski/troms/ver/43254897/
```

### PR DESCRIPTION
**Pull request by TRox1972**

### CODE REVIEW

**Comment by dstftw** [comment]
This embed is generic thus should go to generic extractor.

File: youtube_dl/extractor/dbtv.py [hunk 1]
```diff
@@ -79,3 +79,29 @@ def _real_extract(self, url):
...
+    }
+
+    def _real_extract(self, url):
+        article_id = self._match_id(url)
+        webpage = self._download_webpage(url, article_id)
+
+        iframe_urls = re.findall(r'<iframe src="([^"]+(?:lazy)?player[^"]+)"', webpage)
+
+        if len(iframe_urls) == 1:
+            return self.url_result(self._proto_relative_url(iframe_urls), DBTVIE)
```
Comment (dstftw) [inline youtube_dl/extractor/dbtv.py]: Just always return playlist.

**Comment by TRox1972** [comment]
@dstftw Should I close this and rather add support for it as a generic embed?

**Comment by dstftw** [comment]
Whatever.

**Comment by dstftw** [comment]
What I meant is that there is no matter where you'll implement generic support here or in another PR.

**Pull request closed without merging**